### DISCUSSION TITLE
config.yaml not found or is it found?

### DISCUSSION THREAD

**Discussion by ycsong** [franciscozorrilla/thesis_pipeline#150] [Q&A] [answered]
Hello,

I was able to have metaGEM installed in the internal server in our lab. I am going through the tutorials ([https://github.com/franciscozorrilla/unseenbio_metaGEM](https://github.com/franciscozorrilla/unseenbio_metaGEM) using the set of data that I have in our internal server. I am particularly interested in binning the contigs using metabat.

bash metaGEM.sh -t metabat -j 1 -c 24 -m 80 -h 10
 
I noticed in the standard output messages that config.yaml wasn’t found (although I am sure I set it up before running the code.:
 
```
Proceeding with metabat job(s) ... 
 
Please verify parameters set in the config.yaml file: 
 
paste: config.yaml: No such file or directory
 
Please pay close attention to make sure that your paths are properly configured!
Do you wish to proceed with this config.yaml file? (y/n)y
```
 
I don’t understand the last line of the message, because it seems like it is using config.yaml from somewhere although it says previously that no such file or directory exists.
 
Also, if I were to submit this job to one of the clusters in our server (instead of running it on our log-in node), would this prevent the job from running in interactive mode (i.e. would this circumvent the y/n questions?)

If you could help me clarify these questions, it would be helpful.

Thank you,

Young

**Comment by franciscozorrilla** [comment, marked as answer]
Hey Young,

Based on your error message, it looks like your `config.yaml` file is not present in the same directory as your the `metaGEM.sh` wrapper script. To verify this, simply list the files in your directory `ls -la` or `ll`, as shown in [this section](https://github.com/franciscozorrilla/unseenbio_metaGEM#1-quality-filtering-short-reads-with-fastp) of the tutorial you are referencing.

Thank you for bringing this bug my attention, as it is a small inconsistency that was generated after remodelling the metaGEM repository structure to be compliant with the Snakemake workflow catalogue [rules](https://snakemake.github.io/snakemake-workflow-catalog/?rules=true).

I will fix this issue ASAP and make a note of it in the tutorial, but for now please modify the following line in your `metaGEM.sh` script:
https://github.com/franciscozorrilla/metaGEM/blob/978ae33afb71058e030473c330d699ac62668337/workflow/metaGEM.sh#L270
to:
```
paste ../config/config.yaml
```
This should work for you assuming that your files are in the same structure as the repository files.

Thanks again for opening a discussion on this topic!
Best,
Francisco

**Comment by franciscozorrilla** [comment]
fixed in #151 , thanks again for pointing out this bug :)
